modules/virtualisation/podman/podman.nix

```
@@ -0,0 +1,165 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.podman;
+  toml = pkgs.formats.toml { };
+  json = pkgs.formats.json { };
+
+  inherit (lib) mkOption types;
+
+  podmanPackage = (pkgs.podman.override { inherit (cfg) extraPackages; });
+
+  # Provides a fake "docker" binary mapping to podman
+  dockerCompat = pkgs.runCommandNoCC
+    "${podmanPackage.pname}-docker-compat-${podmanPackage.version}" {
+      outputs = [ "out" "man" ];
+      inherit (podmanPackage) meta;
+    } ''
+      mkdir -p $out/bin
+      ln -s ${podmanPackage}/bin/podman $out/bin/docker
+
+      mkdir -p $man/share/man/man1
+      for f in ${podmanPackage.man}/share/man/man1/*; do
+        basename=$(basename $f | sed s/podman/docker/g)
+        ln -s $f $man/share/man/man1/$basename
+      done
+    '';
+
+in {
+  meta.maintainers = [ lib.maintainers.michaelCTS ];
+  imports = [
+    (lib.mkRenamedOptionModule [ "virtualisation" "podman" "libpod" ] [
+      "virtualisation"
+      "containers"
+      "containersConf"
+    ])
+  ];
+  options.services.podman = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        This option enables Podman, a daemonless container engine for
+        developing, managing, and running OCI Containers on your Linux System.
+
+        It is a drop-in replacement for the <command>docker</command> command.
```

Isn't this true only if `dockerCompat = true`?

---

Podman is a drop in, docker compat just creates an alias binary for script compatibility.

---

Ok, I think it may be my misinterpretation of "drop-in". I was thinking that it would be a drop-in in the sense that if you today have a script that uses the docker command, then after switching you could use the script unchanged with Podman. But it actually is drop-in in the sense that you can use the script with the only change being s/docker/podman/ 🙂

So no need to change, except for the conversion to Markdown
```suggestion
        It is a drop-in replacement for the {command}`docker` command.
```